### DISCUSSION TITLE
简化更新流程

### DISCUSSION THREAD

**Discussion by ValeriaXYN** [MetaCubeX/ClashMetaForAndroid#377] [Ideas]
目前Clash Meta 已经在Play 商店下架，因此唯一的更新方式是：在应用内点击帮助→ Clash Meta for Android →跳转到GitHub 应用或网页→发行版→选择适合的版本下载安装。
能否在应用里内置一个“检查更新“按钮并添加一个“自动下载更新”功能？

**Comment by jihaohaaaa** [comment]
软件在f-droid上可以搜索到，用f-droid更新
